src/bundles/RemoteTutorDrawer/RemoteTutorDrawer.test.tsx

```diff
@@ -0,0 +1,313 @@
+// This was giving false positives
+/* eslint-disable testing-library/await-async-utils */
+import { act, render, screen, waitFor } from "@testing-library/react"
+import user from "@testing-library/user-event"
+import { RemoteTutorDrawer } from "./RemoteTutorDrawer"
+import type { RemoteTutorDrawerInitMessage } from "./RemoteTutorDrawer"
+import { ThemeProvider } from "../../components/ThemeProvider/ThemeProvider"
+import * as React from "react"
+import { http, HttpResponse } from "msw"
+import { setupServer } from "msw/node"
+
+const TEST_API_STREAMING = "http://localhost:4567/test"
+const CONTENT_FILE_URL = "http://localhost:4567/api/v1/contentfiles/1"
+
+const CONTENT_RESPONSE = {
+  count: 1,
+  next: null,
+  previous: null,
+  results: [
+    {
+      id: 1,
+      summary: "This is a test summary",
+      flashcards: [
+        {
+          question: "Test question 1?",
+          answer: "Test answer 1",
+        },
+        {
+          question: "Test question 2?",
+          answer: "Test answer 2",
+        },
+        {
+          question: "Test question 3?",
+          answer: "Test answer 3",
+        },
+      ],
+    },
+  ],
+}
+
+class MockResizeObserver {
+  observe = jest.fn()
+  unobserve = jest.fn()
+  disconnect = jest.fn()
+}
+
+global.ResizeObserver = MockResizeObserver
+
+describe("RemoteTutorDrawer", () => {
+  const server = setupServer(
+    http.post(TEST_API_STREAMING, async () => {
+      return HttpResponse.text("AI Response")
+    }),
+    http.get(CONTENT_FILE_URL, () => {
+      return HttpResponse.json(CONTENT_RESPONSE)
+    }),
+  )
+
+  beforeEach(() => {
+    jest.resetAllMocks()
+  })
+
+  afterEach(() => {
+    jest.restoreAllMocks()
```

Comment:
reset/restore/clear ... So many.

**Anyway:** I don't think this actually does anything. It restores things to the original (non-mocked) implementation, but only if you used `jest.spyOn`.  https://jestjs.io/docs/mock-function-api#mockfnmockrestore